docs/quick-start/configure-mongodb.txt

```diff
@@ -35,7 +35,7 @@ Configure Your MongoDB Connection
          'connections' => [
            'mongodb' => [
              'driver' => 'mongodb',
-             'dsn' => env('DB_URI', '<connection string>'),
+             'dsn' => '<connection string>',
```

Comment:
As the connection string is likely to differ between environments (e.g. dev, staging, prod), we should stick with the Laravel default of providing this in an environment variable.
```suggestion
             'dsn' => env('DB_URI', '<connection string>'),
```